### PR TITLE
safe-rm: added HEAD

### PR DESCRIPTION
**Pull request by kevinawoo**

### CODE REVIEW

File: Library/Formula/safe-rm.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ class SafeRm < Formula
   homepage 'https://launchpad.net/safe-rm'
   url 'https://launchpad.net/safe-rm/trunk/0.10/+download/safe-rm-0.10.tar.gz'
   sha1 '6b829ae68e1fa3c8016e15ab37fcc08caef7712f'
+
```
Comment (DomT4) [inline Library/Formula/safe-rm.rb]: There's whitespace on this line apparently. You'll need to remove it to pass the bot :).

Comment (DomT4) [inline Library/Formula/safe-rm.rb]: Double quotes rather than single quotes too if you're feeling especially kind :smiley_cat:

**Comment by kevinawoo** [comment]
Removed the white space and change single quotes to double. Still isn't passing?

**Comment by tdsmith** [comment]
Rebuilding; we were seeing some bogus failures earlier.

File: Library/Formula/safe-rm.rb [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 class SafeRm < Formula
-  homepage 'https://launchpad.net/safe-rm'
-  url 'https://launchpad.net/safe-rm/trunk/0.10/+download/safe-rm-0.10.tar.gz'
-  sha1 '6b829ae68e1fa3c8016e15ab37fcc08caef7712f'
+  homepage "https://launchpad.net/safe-rm"
+  url "https://launchpad.net/safe-rm/trunk/0.10/+download/safe-rm-0.10.tar.gz"
+  sha1 "6b829ae68e1fa3c8016e15ab37fcc08caef7712f"
+
+  head do
+    url "https://gitorious.org/safe-rm/mainline.git", :branch => 'master'
```
Comment (DomT4) [inline Library/Formula/safe-rm.rb]: The ` branch ` is redundant here I think. ` Master ` is the default AFAIK. Thanks for the double quotes by the way! Appreciate that :+1:

**Pull request closed without merging**